bothub/api/serializers.py

```diff
@@ -223,3 +226,33 @@ class Meta:
             'repository',
             'created_at',
         ]
+
+
+class RegisterUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = [
+            'email',
+            'name',
+            'nick',
```

Comment:
Please use `nickname`

Reply:
I can do this in another PR? This out from PR scope.